app/extensions/brave/content/scripts/passwordManager.js

```diff
@@ -45,11 +45,10 @@ if (chrome.contentSettings.passwordManager == 'allow') {
     }
 
     if (credentials[action]) {
-      credentials[action].push([passwordElem, usernameElem])
-    } else {
-      credentials[action] = [[passwordElem, usernameElem]]
+      return
```

Comment:
if there are mutiple forms on a page with the same action, all of them should be autofilled. i believe this change causes only the first to be autofilled.